src/snowflake/snowpark/functions.py

```diff
@@ -457,6 +465,40 @@ def bitshiftright(to_shift_column: ColumnOrName, n: Union[Column, int]) -> Colum
     return call_builtin("bitshiftright", c, n)
 
 
+def bround(col: ColumnOrName, scale: Union[Column, int]) -> Column:
+    """
+    Rounds the number using `HALF_TO_EVEN` option. The `HALF_TO_EVEN` rounding mode rounds the given decimal value to the specified scale (number of decimal places) as follows:
+    * If scale is greater than or equal to 0, round to the specified number of decimal places using half-even rounding. This rounds towards the nearest value with ties (equally close values) rounding to the nearest even digit.
+    * If scale is less than 0, round to the integral part of the decimal. This rounds towards 0 and truncates the decimal places.
+    NOTE: values are casted to NUMBER(20,8) prior rounding
```

Comment:
I suggest this:

NOTE: Values are casted to NUMBER(20,8) prior rounding.

Reply:
(Casted is technically correct for the past tense, but is more old-fashioned so I suggest using "cast")